CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## 0.18.0 - 2022-06-18
+
+### Added
+
+- Switch from `watchgod` to `watchfiles` (#1437) 18/06/22
```

Comment:
Maybe we can improve this message here? 🤔

Reply:
yes, suggestion below, could be better !